apps/opik-backend/src/main/java/com/comet/opik/api/LlmProvider.java

```diff
@@ -0,0 +1,13 @@
+package com.comet.opik.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum LlmProvider {
+
+    @JsonProperty("openai")
+    OPEN_AI;
+}
```

Comment:
Shouldn't this be like?:

```suggestion
public enum LlmProvider {

    OPEN_AI("openai");
    
    @JsonValue
    private final String value;
}
```

Reply:
I tried this one. It works fine with `ObjectMapper` from `JsonUtils`, but fails to deserialize when sent via POST payload, even it seems that application mapper is configured in the same way. So I chose this implementation which works in both cases.

Reply:
I fixed this issue here: 
- https://github.com/comet-ml/opik/pull/890
As I needed the JSON value in order to return the proper message to users.